policies/trademark-policy.md

```diff
@@ -1,172 +1,325 @@
+- You cannot use our logo on your website in a way that suggests that your website is an official website or that we endorse your website.
+
+These statements could cause confusion when people discover they do not have the same features and functionality they would get if they downloaded the software directly from us, or they believe that someone is endorsed by or affiliated with us when they are not.
+
+What you can say instead:
+
+- You like the Hibernate software
+- You participate in the Hibernate community
+- You are providing an unmodified version of the Hibernate software
+- You wrote a book describing how to use the Hibernate software.
+
+This fundamental requirement, that it is always clear to people what they are getting and from whom, is reflected throughout this Policy.
+It should also serve as your guide if you are not sure about how you are using the Marks.
+
+Trademark law does not allow your use of names or trademarks that are too similar to ours.
+
+- You may not use an obvious variation of any of our Marks or any phonetic equivalent, foreign language equivalent, takeoff, or abbreviation for a similar or compatible product or service.
 
-A copyright license, even an open source copyright license, does not include an implied right or license to use a trademark that may be related to the project developing the licensed software or other materials.
-Because open source licenses permit unrestricted modification of the copyrighted software,
-and the Commonhaus Foundation has no intention of interfering with that right,
-your right to use any specific trademark of the Commonhaus Foundation is not determined by your use of software made available under an open source license.
-Your right to use a Mark of the Commonhaus Foundation is provided for in this policy and in the statement of permitted use, if any, that may accompany the trademark notice displayed on a CF project website.
+- You may not use or register, in whole or in part, the Marks as part of your own trademark, service mark, domain name, company name, trade name, product name or service name.
 
-Proper use of Commonhaus Foundation trademarks by following these trademark usage guidelines protects the value of the Commonhaus Foundation trademarks.
-Any use of or reference to the Commonhaus Foundation trademarks that is inconsistent with this trademark policy, or use of marks that are confusingly similar to trademarks of the Commonhaus Foundation, is prohibited.
+In addition:
 
-All uses of Commonhaus Foundation trademarks, and all goodwill associated therewith, will inure solely to the benefit of the Commonhaus Foundation.
+- You may not use the Marks in association with the use or distribution of software if you are also not in compliance with the copyright license for the software.
 
-### Acknowledgment and Symbols
+- You agree that you will not acquire any rights in the Marks and that any goodwill generated by your use of the Marks inures solely to our benefit.
```

Comment:
"any goodwill generated by your use of the Marks inures solely to our benefit"

why is this necessary? i can't guarantee whose benefit our use of the mark inures to.

Reply:
"goodwill" in this context has a legal meaning. I'm thinking of adding this as a footnote to help explain:

```suggestion
- You agree that you will not acquire any rights in the Marks and that any goodwill generated by your use of the Marks inures solely to our benefit.[^1]

[^1]: This clause prevents someone who uses the trademark from later claiming that, because their use of the mark built up goodwill (meaning the brand’s reputation and value) for them, they own the trademark. By including this statement, it makes clear that any goodwill generated from using the mark belongs to the Foundation (and the Project), not the user (licensee). It also serves as evidence that the user agreed not to claim ownership of the mark or its associated goodwill.
```